src/avitab/apps/MapApp.h

```diff
@@ -37,9 +37,28 @@
 #include "src/maps/OverlayedMap.h"
 #include "src/maps/sources/NavigraphSource.h"
 #include "src/maps/sources/OnlineSlippyMapConfig.h"
+#include "src/maps/sources/OnlineSlippySource.h"
 
 namespace avitab {
 
+const struct maps::OnlineSlippyMapConfig fallbackOnlineMap = {

```

Comment:
Here is the hardcoded fallback - even if the mapconfig.json is broken, this will still work